sdk/formrecognizer/Azure.AI.FormRecognizer/tests/DocumentAnalysisClient/DocumentAnalysisClientLiveTests.cs

```diff
@@ -1570,6 +1622,15 @@ private void ValidateAnalyzeResult(AnalyzeResult result, string modelId, int exp
             {
                 ValidateTable(table, expectedFirstPageNumber, expectedLastPageNumber);
             }
+
+            // Check Document Languages.
```

Comment:
I noticed we're not validating spans at all in our tests. Created https://github.com/Azure/azure-sdk-for-net/issues/26766 for tracking.